### ISSUE TITLE
implement with pytorch

### ISSUE THREAD

**Issue by jzhijun** [Cybernetic1/policy-gradient#1]
hello,very happy to talk with you. you work is very meaningful. if you do it with pytorch, it's  better.

**Comment by Cybernetic1** [comment]
Yes, thanks for your comment, and indeed I am planning to switch to PyTorch :)

**Comment by Cybernetic1** [comment]
If anyone wants to help or collaborate, please contact me or leave a message.
I will start with copying a simple implementation of policy gradient with PyTorch and work from there...
